router/src/main/resources/Dockerfile

```diff
@@ -47,6 +47,10 @@ ENV ROUTER_LABEL="Default"
 ENV ROUTER_PRIVATE_KEY_PATH=/home/wso2/security/keystore/mg.key
 ENV ROUTER_PUBLIC_CERT_PATH=/home/wso2/security/keystore/mg.pem
 
+ENV KEEPALIVE_PROBES=3
+ENV KEEPALIVE_TIMEOUT=60
```

Comment:
Shall we keep the current default as it is and identify a proper value in a lower environment before sending this to higher environments.

Reply:
Reverted values in ac33a9c33082062051a8f6c5f2685e743f408b0b